migrations/entitlements/migration.go

```diff
@@ -254,10 +261,14 @@ func ConvertValueToEntitlements(
 			return nil
 		}
 
-		convertedType, _ := ConvertToEntitledType(
+		convertedType, converted := ConvertToEntitledType(
 			inter.MustConvertStaticToSemaType(v.Type),
 		)
 
+		if !converted {
+			return nil
+		}
```

Comment:
If the type is not converted, it returns a `nil`. So exit early here, and avoid wrapping `nil` with a type value.